cg/meta/archive/archive.py

```diff
@@ -40,33 +35,26 @@ def filter_files_on_archive_location(
     ]
 
 
+archive_handlers: Dict[str, Callable] = {ArchiveLocations.KAROLINSKA_BUCKET: DDNDataFlowClient}
```

Comment:
THis would be a similar structure as to what we use in cg when we made the store filters. Maybe this can be done here also. It would be easily extendable and we do not need to rely on dicts.

The class name can be changed just took the variable name that was given

```suggestion
class ArchiveHandlers(Enum):
KAROLINSKA_BUCKET: Callable = DDNDataFlowClient
```

Reply:
Seems a bit cumbersome since you can not reuse enums within another enum, and for an enum to have multiple values, the easiest way would be to add "aenum" to the requirements. Would that be something we would want?

Reply:
Seems as if a common way is to use a dict in these instances if we don't want to import aenum, so I will keep as is for now. https://stackoverflow.com/questions/70566944/python-enum-keys-from-another-enum